src/components/WalletKit/ConnectedModal.tsx

```diff
@@ -46,21 +45,11 @@ export const ConnectedModal = ({
       <Modal.Footer itemAlignment="stack">
         <Button size="md" variant="tertiary">
           <CopyText textToCopy={publicKey}>Copy address</CopyText>
-          <IconButton
-            altText="Default"
-            customColor="var(--sds-clr-gray-09)"
-            customSize="14px"
-            icon={<Icon.Copy01 />}
-          />
+          <Icon.Copy01 />
         </Button>
```

Comment:
We can add an icon through the `Button` component's props: `icon` (pass the icon) and `iconPosition` (`left` or `right`, defaults to `right`). Same for the button below.

It might be better if we wrapped the `Button` component to make the whole `Button` area clickable.

```typescript
<CopyText textToCopy={tx.txHash}>
  <Button
    size="sm"
    variant="tertiary"
    icon={<Icon.Copy01 />}
    iconPosition="left"
    onClick={(e) => e.preventDefault()}
  >Copy address</Button>
</CopyText>
```